### ISSUE TITLE
Ensure that the color code is readable on dark color background.

### ISSUE THREAD

**Issue by uvisgrinfelds** [sdushantha/farge#8]
Love the tool!

Now if you color pick a black(or dark) color, the hex code is not readable. The script should detect the contrast ratio, and change the text color accordingly.

**Comment by sdushantha** [comment]
@uvisgrinfelds That would be a great idea!
But I have no clue on how to detect if the color is dark or light. It might take some time before this feature comes.

**Comment by sdushantha** [comment]
If you know how to do this, please feel free to create a PR :)

**Comment by sdushantha** [comment]
@uvisgrinfelds Fixed this!

**Comment by uvisgrinfelds** [comment]
Awesome, thanks!

> @uvisgrinfelds Fixed this!
